cli/plasmo/src/features/manifest-factory/base.ts

```diff
@@ -320,24 +322,51 @@ export abstract class BaseFactory<T extends ExtensionManifest = any> {
     return enable
   }
 
-  addContentScriptsDirectory = async (contentsDirectory: string) => {
-    const hasContentsDirectory = existsSync(contentsDirectory)
-    if (!hasContentsDirectory) {
+  addDirectory = async (
+    dPath: string,
```

Comment:
since the name is `addDirectory` we can just write `path` here since the func is about directories